Dockerfile.QA

```diff
@@ -99,16 +99,18 @@ RUN mkdir -p qa/common && \
     cp -r docs/examples/model_repository/simple qa/L0_cuda_shared_memory/models/. && \
     mkdir qa/L0_client_java/models && \
     cp -r docs/examples/model_repository/simple qa/L0_client_java/models && \
-    mkdir qa/L0_grpc/models && \
+    mkdir qa/L0_grpc/models && mkdir qa/L0_grpc/unit_test_models && \
     cp -r docs/examples/model_repository/simple qa/L0_grpc/models && \
+    cp -r docs/examples/model_repository/simple qa/L0_grpc/unit_test_models && \
     cp -r docs/examples/model_repository/simple_dyna_sequence qa/L0_grpc/models && \
     cp -r docs/examples/model_repository/simple_int8 qa/L0_grpc/models && \
     cp -r docs/examples/model_repository/simple_identity qa/L0_grpc/models && \
     cp -r docs/examples/model_repository/simple_sequence qa/L0_grpc/models && \
     cp -r docs/examples/model_repository/simple_string qa/L0_grpc/models && \
     cp -r docs/examples/model_repository/inception_graphdef qa/L0_grpc/models && \
-    mkdir qa/L0_http/models && \
+    mkdir qa/L0_http/models && mkdir qa/L0_http/unit_test_models && \
     cp -r docs/examples/model_repository/simple qa/L0_http/models && \
+    cp -r docs/examples/model_repository/simple qa/L0_http/unit_test_models && \
```

Comment:
You don't need to create the directory in Dockerfile, it can be part of the test script, i.e. in line 472 of test.sh:
```
rm -rf unit_test_models
mkdir unit_test_models && cp -r $DATADIR/qa_model_repository/onnx_int32_int32_int32 unit_test_models/. &&  cp -r ${MODELDIR}/simple unit_test_models/. 
```

Reply:
And
```
SERVER_ARGS="... --model-repository=unit_test_models ..."
```

Reply:
Thanks, updated.